scripts/service_migration.py

```diff
@@ -0,0 +1,89 @@
+import copy
+from kubernetes import client, config
+from kubernetes.client.exceptions import ApiException
+import argparse
+import os.path
+from pprint import pprint
+
+IP_ANNOTATION = 'load-balancer-ips'
+def update_annot(v1, svc):    
```

Comment:
update_annotation will be more readable I think.

Reply:
Updated function name